app/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,71 @@
+<resources>
+    <string name="app_name">DroidKaigi 2016</string>
+    <string name="tag_line">المهندسين الروبوت من مؤتمر بطل الرواية</string>
+    <string name="session_detail">تفاصيل الدورة</string>
```

Comment:
These Arabic text are translated by Google translate. So it is not correct at all...
This PR is just for my session of DroidKaigi :bow: